Index: lib/getopt.c
--- gnulib/lib/getopt.c	Thu Mar 11 15:04:48 2004
+++ lib/getopt.c	Sun May 16 12:34:12 2004
@@ -48,7 +48,7 @@
 # endif
 #endif
 
-#ifndef ELIDE_CODE
+#if defined WANT_INCLUDED_GETOPT || !defined ELIDE_CODE
 
 
 /* This needs to come after some library #include
Index: lib/getopt1.c
--- gnulib/lib/getopt1.c	Tue Mar  9 21:42:23 2004
+++ lib/getopt1.c	Sun May 16 12:34:36 2004
@@ -46,7 +46,7 @@
 #endif
 #endif
 
-#ifndef ELIDE_CODE
+#if defined WANT_INCLUDED_GETOPT || !defined ELIDE_CODE
 
 
 /* This needs to come after some library #include
